adal/src/main/java/com/microsoft/aad/adal/TokenCacheAccessor.java

```diff
@@ -381,10 +381,10 @@ private void setItemToCacheForUser(final String resource, final String clientId,
      */
     private List<String> getKeyListToRemoveForRT(final TokenCacheItem cachedItem) {
         final List<String> keysToRemove = new ArrayList<>();
-        keysToRemove.add(CacheKey.createCacheKeyForRTEntry(mAuthority, cachedItem.getResource(), cachedItem.getClientId(), null));
+        keysToRemove.add(CacheKey.createCacheKeyForRTEntry(cachedItem.getAuthority(), cachedItem.getResource(), cachedItem.getClientId(), null));
```

Comment:
Quick question: if this fix is desired, do we also need to change getKeyListToRemoveForMRRT(...) and getKeyListToRemoveForFRT(...)?

Reply:
Yes, we should also do the same change with MRRT and FRT. This PR was paused from last week.